ci/common.rb

```diff
@@ -42,6 +42,26 @@ def travis_pr?
   !ENV['TRAVIS'].nil? && ENV['TRAVIS_EVENT_TYPE'] == 'pull_request'
 end
 
+BAD_CITIZENS = {
+  'couch' => 'couchdb',
+  'disk' => 'sysstat',
```

Comment:
this specific exception is confusing IMO, and could be avoided... How about we add a `disk` flavor to the travis test matrix instead?